modules/claims_api/lib/claims_api/v2/disability_compensation_evss_mapper.rb

```diff
@@ -0,0 +1,121 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  module V2
+    class DisabilityCompensationEvssMapper
+      def initialize(auto_claim)
+        @auto_claim = auto_claim
+        @data = auto_claim&.form_data&.deep_symbolize_keys
+        @evss_claim = {}
+      end
+
+      def map_claim
+        claim_attributes
+        claim_meta
+
+        { form526: @evss_claim }
+      end
+
+      private
+
+      def claim_attributes
+        change_of_address
+        current_mailing_address
+        direct_deposit
+        disabilities
+        service_information
+        standard_claim
+        veteran_meta
+      end
+
+      def change_of_address
+        return if @data[:changeOfAddress].blank?
+
```

Comment:
Looking at https://jira.devops.va.gov/browse/API-26640 it looks like we only need veteranIdentification, not ChangeOfAddress.